test/Microsoft.Health.Dicom.Tests.Integration/Persistence/ExtendedQueryTagErrorStoreTests.cs

```diff
@@ -103,6 +108,36 @@ public async Task GivenExistingExtendedQueryTagandTagError_WhenDeleteExtendedQue
             Assert.Equal(0, extendedQueryTagAfterTagDeletion.Count);
         }
 
+        [Fact]
+        public async Task GivenExistingInstanceandExtendedQueryTagandTagError_WhenDeleteInstance_ThenTagErrorShouldAlsoBeRemoved()
```

Comment:
Can you also write a test where you delete a study or series with more than 1 instance?